### PR TITLE
filteraudio-git: fetching code via proper distfile

### PR DESCRIPTION
**Pull request by teh-jazzman**
In answer to #2070

### CODE REVIEW

File: srcpkgs/filteraudio-git/template [hunk 1]
```diff
@@ -1,17 +1,16 @@
 # Template file for 'filteraudio-git'
 pkgname="filteraudio-git"
-version="20150313"
-revision=1
+version="20150722"
+revision=2
```
Comment (beefcurtains) [inline srcpkgs/filteraudio-git/template]: Good work, but bumping both version _and_ revision seems redundant to me.

Comment (teh-jazzman) [inline srcpkgs/filteraudio-git/template]: That just happend. I first only updated the version and tested it but later made some other changes and increased the revision.

Comment (beefcurtains) [inline srcpkgs/filteraudio-git/template]: Please? :smile_cat:

**Comment by thypon** [comment]
Alredy fixed.

**Pull request closed without merging**